server/views/referrals/new/courseParticipations/index.njk

```diff
@@ -52,23 +31,47 @@
             "data-testid": "success-banner"
           }
         }) }}
-      {% endif %}
+      {% else %}
+        {{ govukInsetText({
+          classes: "govuk-!-margin-top-0",
+          text: historyText,
+          attributes: {
+            "data-testid": "history-text"
+          }
+        }) }}
+
+        {% if summaryListsOptions.length %}
+          <p class="govuk-body" data-testid="pre-history-paragraph">Add another programme if you know that he started or completed a programme which is not listed below or skip this section of the referral if the history is not known.</p>
+        {% else %}
+          <p class="govuk-body" data-testid="no-history-paragraph">The programme team may use information about a person's Accredited Programme history to assess whether they are suitable.</p>
 
-      {% if summaryListsOptions.length %}
-        <p class="govuk-body" data-testid="pre-history-paragraph">The history below shows {{ person.name }}'s history of Accredited Programmes. Add another programme if you know that they started or completed a programme which is not listed below.</p>
+          <p class="govuk-body" data-testid="no-history-paragraph">You can continue by adding a programme history or skip this section of the referral if the history is not known.</p>
```

Comment:
I see that in the tests you're using `eq` to get a specific `no-history-paragraph`, but would it be better to use unique numbered IDs (given they're IDs)?